.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -1,43 +1,48 @@
 ---
 name: '🐛 Bug Report'
-about: Report a reproducible bug or regression.
+about:  Create a report to help us improve
```

Comment:
Remove period for consistency

<img width="432" alt="Screenshot 2023-11-08 at 11 22 25 AM" src="https://github.com/wpengine/faustjs/assets/6676674/bd282919-55f6-41b2-b159-aa7c86c5999c">